src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/SmoTriggerCustomNode.cs

```diff
@@ -12,10 +14,27 @@ namespace Microsoft.SqlTools.ServiceLayer.ObjectExplorer.SmoModel
     /// </summary>
     internal partial class TriggersChildFactory : SmoChildFactoryBase
     {
+        public static readonly List<NodeSmoProperty> SmoPropertyList = new List<NodeSmoProperty>
```

Comment:
Minor concern: Slightly better to make this lazily initialized for two reasons. Both are not big concerns for small objects like this one.
1. Static object constructs increase process startup time/load time.
2. If no one ever expands the triggers node then this is not constructed